node/pkg/processor/cleanup.go

```diff
@@ -228,8 +228,11 @@ func (p *Processor) handleCleanup(ctx context.Context) {
 					if err := common.PostObservationRequest(p.obsvReqSendC, req); err != nil {
 						p.logger.Warn("failed to broadcast re-observation request", zap.String("message_id", s.LoggingID()), zap.Error(err))
 					}
-
-					p.gossipAttestationSendC <- s.ourMsg
+					if s.ourMsg != nil {
+						p.gossipAttestationSendC <- s.ourMsg
```

Comment:
Seems like this will potentially re-send un-batched observations after the cutover - I think that's probably okay as long as folks are aware of this possibility (that both will need to be supported until after `retryLimitOurs` which is currently 30 hours) - but just wanted to double-check.

Reply:
That is correct, after the cutover any retries on old messages will still use the old message format. As long as we don't immediately rip out the old code, that should be fine.

Additionally, this case also covers the immediately published messages, so this code will have to remain when we rip out the cutover stuff. I added a comment about that.